pkg/payload/render.go

```diff
@@ -22,7 +22,10 @@ func Render(outputDir, releaseImage string) error {
 		bootstrapDir  = "/bootstrap"
 		oBootstrapDir = filepath.Join(outputDir, "bootstrap")
 
-		renderConfig = manifestRenderConfig{ReleaseImage: releaseImage}
+		renderConfig = manifestRenderConfig{
+			ReleaseImage:   releaseImage,
+			ClusterProfile: defaultClusterProfile,
+		}
```

Comment:
I am not sure who calls this function? Is it installer during bootstrap?

Reply:
I mean `Render(outputDir, releaseImage string) error `

Reply:
Yes, this code https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L62

Reply:
The env. variable is not plugged yet. This PR just teaches enough to the CVO for updating to a CVO that uses ClusterProfile in his manifests.

Reply:
Looks like the the PR for env variable already merged for installer.